packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -636,7 +705,6 @@ export const previewActionReceived: Action<{
       actions.editor.addNpmDependency({
         name,
       });
-      actions.forceRender();
```

Comment:
No need?

Reply:
Nopes `forceRender` was used to trigger the preview, but now trigger preview imperatively through the `effects.preview` API 😄